tensorflow_probability/python/positive_semidefinite_kernels/BUILD

```diff
@@ -122,3 +122,26 @@ py_test(
         "//tensorflow_probability",
     ],
 )
+
+py_library(
+    name = "dot_product",
```

Comment:
Prefer to name kernels after their class name (so linear in this case).

Reply:
renamed to "polynomial" to name it after kernel (made the linear kernel more general by adding an optional exponent term)